fabric-networking-v0/src/main/java/net/fabricmc/fabric/mixin/networking/MixinPlayerManager.java

```diff
@@ -38,10 +38,9 @@
 	public void onPlayerConnect(ClientConnection connection, ServerPlayerEntity player, CallbackInfo info) {
 		Optional<Packet<?>> optionalPacket = PacketRegistryImpl.createInitialRegisterPacket(ServerSidePacketRegistry.INSTANCE);
 
-		//noinspection OptionalIsPresent
-		if (optionalPacket.isPresent()) {
-			player.networkHandler.sendPacket(optionalPacket.get());
+		optionalPacket.ifPresent(packet -> {
+			player.networkHandler.sendPacket(packet);
 			((ServerSidePacketRegistryImpl) ServerSidePacketRegistry.INSTANCE).addNetworkHandler(player.networkHandler);
-		}
+		});
```

Comment:
If you want to take the lambda route, you may as well remove the optional variable and directly chain calls